config/manager/manager.yaml

```diff
@@ -34,5 +34,15 @@ spec:
             valueFrom:
               fieldRef:
                 fieldPath: metadata.annotations['olm.targetNamespaces']
+          - name: BUILDER_IMAGE
```

Comment:
do we really need those values here as they are the default ones ?
Or are they for documentation purpose ?

Reply:
> btw is there a way to update those values in OLM ?

Not sure. I imagine we will hard code the default image names in the product CSV.

Reply:
for community operator, these values are not required because these env var default values are defines as per community operator requirement. So if remove these env var from deployment in community operator, then it will pick its default values and works fine. But these value will mandatory to define product operator because default values are invalid in case of product operator.

For now, its work as sample for How we should define these value in product operator.

Reply:
ok fine for em